### PR TITLE
Upgrade to ES 7.16.2

### PR DESCRIPTION
**Pull request by eladiw**
The following changes are proposed:

* Upgraded to ES 7.16.2
* Added new field caps field [metadata_field](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-fields.html)
* fixed tests

### CODE REVIEW

**Review by tomconte — APPROVED** [review]

File: charts/k2bridge/values.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ elasticsearch:
   clusterName: k2bridgees
   masterService:  k2bridgees-master
 
-  image: docker.elastic.co/elasticsearch/elasticsearch-oss
-  imageTag: 7.10.2
+  image: docker.elastic.co/elasticsearch/elasticsearch
+  imageTag: 7.16.2
```
Comment (tamirkamara) [inline charts/k2bridge/values.yaml:43]: Is it perhaps the right time to also update the elastic chart version we use?

Comment (AsafMah) [reply charts/k2bridge/values.yaml:43]: what's elastic chart?

**Review by AsafMah — CHANGES_REQUESTED** [review]
I understand the version changes, but why add metadata_field if it's always false?

**Comment by eladiw** [comment]
> I understand the version changes, but why add metadata_field if it's always false?

to match the results ES returns.
maybe Kibana use it, maybe not.
In anycase the e2e failed, since now our reference ES instance returns it and K2 didn't.

File: charts/k2bridge/values.yaml (hunk 1 above)
Comment (eladiw) [reply charts/k2bridge/values.yaml:43]: updated.

@AsafMah , its the chart the defines how to deploy elastic. we use it as a dependency
https://github.com/elastic/helm-charts/tree/main/elasticsearch

**Review by AsafMah — APPROVED** [review]

**Pull request merged**